### ISSUE TITLE
IEMOCAP download link does not contain iemocap_data.pkl

### ISSUE THREAD

**Issue by clionNJ** [taco-group/DecAlign#10]
Hi authors, thanks for releasing DecAlign!

I’m trying to reproduce the IEMOCAP experiments following the README. The README states that the required file is iemocap_data.pkl, but the Google Drive link only contains:

IEMOCAP_features_raw_4way.pkl
IEMOCAP_features_raw_6way.pkl
There is no file named iemocap_data.pkl in the provided link.
I tried renaming the raw feature files, but they are not compatible with the expected data format in .data_loader.py

Could you please:

Upload/provide the correct iemocap_data.pkl as referenced in the README, or
Provide a conversion/preprocessing script or instructions to transform the raw feature files into the expected format?
Thanks a lot!

**Comment by Raymond-Qiancx** [comment]
Hi, thanks for raising this issue.

We re-checked the released IEMOCAP data link and confirmed that the shared files are raw feature files rather than the directly loadable `iemocap_data.pkl` expected by the training code.

To make the preprocessing step explicit, we have added an IEMOCAP preparation script in the updated codebase. The script converts the provided raw IEMOCAP feature file into the format required by DecAlign:

```bash
python scripts/prepare_iemocap.py \
  --data_dir /path/to/data
```

The expected output is:

```text
/path/to/data/IEMOCAP/iemocap_data.pkl
```

The generated pickle follows the DecAlign dataloader format and contains `train`, `valid`, and `test` splits with text, audio, vision features, raw text, utterance ids, and classification labels.

In our updated setup, the IEMOCAP config points to:

```text
IEMOCAP/iemocap_data.pkl
```

and uses the pre-extracted features directly, so BERT is disabled for IEMOCAP.

Please pull the latest update and run the preparation script before launching IEMOCAP training. Thanks again for pointing this out.

**Comment by sun-xiao-pan** [comment]
Hi authors,
I couldn't find the file you mentioned, scripts/prepare_iemocap.py. Could you add this file again? 
Thanks a lot!
